Modules/_zstd/decompressor.c

```diff
@@ -121,7 +121,7 @@ _PyZstd_set_d_parameters(ZstdDecompressor *self, PyObject *options)
 
 /* Load dictionary or prefix to decompression context */
 int
```

Comment:
```suggestion
static int
```

See above about static linkage.